AI-Upscaling-With-NPU/utils.py

```diff
@@ -221,6 +228,13 @@ def download_file(url, output_file):
 
 
 def resize_video(input_path, scale=2):
+    """resizes a video by a given scale factor
```

Comment:
That function decreases size of video. Could you write it here? It`s a bit unintuitive behaviour

Reply:
Done, I think now it is clearly said what this function does.